src/util/AutoFetchMore.ts

```diff
@@ -0,0 +1,10 @@
+import { MutableRefObject } from "react";
+
+export function AutoFetchMore(
```

Comment:
Is there a reason why this function starts with a capital letter?
I think it should start with small

Reply:
I will change the name totally